### ISSUE TITLE
Is it possible render template without $data param?

### ISSUE THREAD

**Issue by shinriyo** [nickel-org/nickel.rs#312]
Hi there.

I always watch sample `render` method with `$data`
but, I can't find wihtoout `$data` param.

```
            return res.render("src/templates/main.tpl");
```
is best.
but, `render` method's 2nd param is not optional

So, I always set dummy data....
```
            let mut data = HashMap::<&str, &str>::new();
            data.insert("dummy", "dymmy");
            return res.render("src/templates/main.tpl", &data);
```
However it isn't possible `return res.render("src/templates/main.tpl", &[[);`

**Comment by SimonPersson** [comment]
If your template does not need any data, then it's just plain HTML, no? If so, just serve that file statically, `res.send_file("path/to/main.html")`. Or is there some use case for templates without data that I'm not aware of?

**Comment by shinriyo** [comment]
@SimonPersson 
Thank you for replying.
yes, I will render just plain HTML. The template doesn't need &data parameters.
I tried your `send_file()` method. but, It makes downloading main.html file.

I guess  I will use the case is home page(like  index.tpl).

**Comment by Ryman** [comment]
> I tried your send_file() method. but, It makes downloading main.html file.

I think setting the `ContentType` header to html should fix this? e.g. `response.set(MediaType::Html);`

**Comment by cburgdorf** [comment]
Yep, but it's quite inconvenient. I know I ran into this myself before and we had a [discussion](https://github.com/nickel-org/nickel.rs/pull/257) on exactly that. Not sure about the right way forward but there should be a simple way to render a template without data, without explicitly setting the `MediaType::Html`.

**Comment by shinriyo** [comment]
@Ryman 
Thank you for replying.
I resolved it.

@cburgdorf 
Did you discussed about it, didn't you?
yes, I hope simple way.
Or, if Rust language and nickel.rs have slogan, TOOWTD, I give up.
